### ISSUE TITLE
Go back link

### ISSUE THREAD

**Issue by tomhub** [lexjansen/define-xml-2.0-stylesheets#7]
I was wondering if "Go back" link before "Go to the top.." would ease navigation. Eg., going from the dataset table to the codelist table, and back one could go back by clicking "back" button on the browser, or there could be a link to go back one history:

```
  <!-- ************************************************************* -->
  <!-- Link to Top                                                   -->
  <!-- ************************************************************* -->
  <xsl:template name="linkTop">
    <p class="linktop">Go <a href="javascript:history.back()">back</a>. Go to the <a href="#main">top</a> of the Define-XML document</p>
  </xsl:template>
```

**Comment by lexjansen** [comment]
Isn't that the same as the regular browser back button?

**Comment by tomhub** [comment]
Eg. when a link to a codelist is clicked, usually a mouse pointer is somewhere nearer to a screen centre, or the right side of a screen. And when the "longer" codelist table is scrolled down, a "Go to the top" link is usually much closer the mouse pointer and viewers attention than the browsers back button. Talking here from my own experience.

"Go to the top" is also achievable with browsers interface using scroll bars or home button on keyboard.

**Comment by lexjansen** [comment]
Yes, true, "Go to the top" can also be achieved with the native browser interface or keys. I think the main reason that it is there is simply because it has always been there in previous stylesheet versions.
We can think of many other navigation buttons. Go to previous codelist, go to next codelist, etc etc...
